src/misc.ts

```diff
@@ -32,10 +32,14 @@ export type PartialOrAbsent<Value> = Partial<Value> | null | undefined;
 export type NonEmptyArray<Element> = [Element, ...Element[]];
 
 /**
- * A JavaScript object that is not `null`, a function, or an array. The object
- * can still be an instance of a class.
```

Comment:
Oops, this deletion was intended to be in #78. The tests added there show that instances of a class were not compatible with the `RuntimeObject` type.